src/tracing.rs

```diff
@@ -57,6 +57,39 @@ impl SpanProcessor for MetricsSpanProcessor {
             ])
             .collect::<Vec<_>>();
 
+        let code = span
+            .attributes
+            .iter()
+            .find(|attr| attr.key.as_str() == "code")
+            .and_then(|attr| attr.value.as_str().parse::<i32>().ok())
+            .unwrap_or(0);
+
+        if code != 0 {
+            counter!("error", &labels).increment(1);
```

Comment:
is there a need for the counter here? the code should be included in `histogram!(format!("{}_duration", span.name), &labels).record(duration);`

Reply:
no need to have the code here